chia/wallet/wallet_node.py

```diff
@@ -81,27 +82,27 @@ def get_wallet_db_path(root_path: Path, config: Dict[str, Any], key_fingerprint:
 
 @dataclasses.dataclass
 class WalletNode:
-    config: Dict
+    config: Dict[str, Any]
     root_path: Path
     constants: ConsensusConstants
     local_keychain: Optional[Keychain] = None
 
     log: logging.Logger = logging.getLogger(__name__)
 
     # Normal operation data
-    cached_blocks: Dict = dataclasses.field(default_factory=dict)
-    future_block_hashes: Dict = dataclasses.field(default_factory=dict)
+    # cached_blocks: Dict = dataclasses.field(default_factory=dict)
```

Comment:
@almogdepaz Do we intend to use these commented out variables?
```
cached_blocks
future_block_hashes
proof_hashes
```

Reply:
good question but i have no idea what they are for, github says @altendky was the one that added these

Reply:
I was just refactoring to a dataclass in https://github.com/Chia-Network/chia-blockchain/commit/75115f2475e1e5f2421bccb8fbb80f7bafe93740.  In `.__init__()` there was `self.cached_blocks: Dict = {}` so i added `cached_blocks: Dict = dataclasses.field(default_factory=dict)`.  Other uses for `cached_blocks` at least were removed prior to mainnet https://github.com/Chia-Network/chia-blockchain/blame/88cf51ad82b6cddf4e47b26f50de54729db09be5/src/wallet/wallet_node.py.  Likewise for `future_block_hashes` and `proof_hashes`.  Well, at least within `src/wallet/wallet_node.py`.

Reply:
Okay, sounds like it is safe for me to remove these. Thanks!